OrbitCore/CMakeLists.txt

```diff
@@ -148,17 +151,17 @@ if(WIN32)
 else()
   target_sources(
     OrbitCore
-    PUBLIC BpfTraceVisitor.h
-           BpfTrace.h
-           LibunwindstackUnwinder.h
-           LinuxEventTracer.h
-           LinuxPerfEvent.h
-           LinuxPerfEventProcessor.h
-           LinuxPerfEventVisitor.h
-           LinuxPerfRingBuffer.h
-           LinuxPerfUtils.h
-           LinuxUprobesUnwindingVisitor.h
-           LinuxUtils.h)
+    PUBLIC ${HEADERS}/BpfTraceVisitor.h
+           ${HEADERS}/BpfTrace.h
+           ${HEADERS}/LibunwindstackUnwinder.h
+           ${HEADERS}/LinuxEventTracer.h
+           ${HEADERS}/LinuxPerfEvent.h
+           ${HEADERS}/LinuxPerfEventProcessor.h
+           ${HEADERS}/LinuxPerfEventVisitor.h
+           ${HEADERS}/LinuxPerfRingBuffer.h
+           ${HEADERS}/LinuxPerfUtils.h
+           ${HEADERS}/LinuxUprobesUnwindingVisitor.h
+           ${HEADERS}/LinuxUtils.h)
```

Comment:
Maybe it is better not to hide these behind a variable... Ideally replace with something like everything under include/ dir? does cmake support this?